.github/workflows/validate_pull_request.yml

```diff
@@ -10,12 +10,16 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: | 
+          dotnet-version: |
             7.0.304
             6.0.300
-      - run: |
-          dotnet build CSharpier.sln -c release
-          dotnet test CSharpier.sln -c release
+      - run: >
+          dotnet test CSharpier.sln
+          --configuration Release
+          --logger "GitHubActions;summary.includePassedTests=true;summary.includeSkippedTests=true"
+          --
+          RunConfiguration.CollectSourceInformation=true
+
```

Comment:
I removed `dotnet build` here because `dotnet test` runs it implicitly anyway. This will save some time.